text/0000-project-error-handling.md

```diff
@@ -0,0 +1,137 @@
+- Produce learning resources related to current best practices.
+    - New chapters in the book?
+
+### Evaluate options for error reporting type a.k.a. better `Box<dyn Error>`
+
+- Survey the current libraries in the ecosystem:
+    - `anyhow`
+    - `eyre`
+- Evaluate value of features including:
+    - Single word width on stack
+    - Error wrapping with display types and with special downcast support.
+    - Report hook and configurable `dyn ReportHandler` type for custom report formats and content, similar to panic handler but for errors.
+    - `#[no_std]` support
+
+### Consolidate ecosystem by merging best practice crates into std
+
+- Provide a derive macro for `Error` in std.
+- Stabilize the `Backtrace` type but possibly not `fn backtrace` on the `Error` trait.
+    - Provide necessary API on `Backtrace` to support crates like `color-backtrace`.
+- Move `Error` to core.
+    - Depends on generic member access.
+    - Requires resolving downcast dependency on `Box` and blocking the stabilization of `fn backtrace`.
+- Potentially stabilize an error reporting type based on `anyhow` and `eyre` now that they're close to having identical feature sets.
+
+### Add missing features
+
+- Generic member access on the `Error` trait.
+- `Error` return traces:
+    - Depends on specialization and generic member access.
+- Fix rough corners around reporting errors and `Termination`.
+
+## Non Goals
+
+- This group should not be involved in managing design discussions for the `Try` trait, `try` blocks, or `try` fns.
+
+## Membership Requirements
+
+- Group membership is open, any interested party can participate in discussions, repeat contributors will be added to appropriate teams.
+
+## Additional Questions
+
+### What support do you need, and separately want, from the Rust organization?
+
+I'm not sure, my main concern is getting prompt feedback on RFCs.
+
+### Why should this be a project group over a community effort?
+
+There isn't anything in this project group that can't be handled as a
+community effort, but centralizing work into a project group should help
+speed things. Error handling is a core aspect of the language and changes in
+error handling have large impacts on the ecosystem. Ensuring that efforts to
+refine error handling within Rust have sufficient resources and don't stall
+out is in the best interests of the community. By organizing efforts as a
+project group we will hopefully have an easier time recruiting new members,
+getting attention on RFCs from members of the libs team, and using the
+established resources and expertise of the rust organization for coordinating
+our efforts.
+
+### What do you expect the relationship to the team be?
+
+The project group will create RFCs for various changes to the standard library and the team will review them via the standard RFC process.
+
+### Who are the initial shepherds/leaders? (This is preferably 2–3 individuals, but not required.)
+
+Jane Lusby, Andrew Gallant, and Sean Chen.
```

Comment:
I'd like to suggest including the relevant GitHub usernames here.  (I'm afraid I don't know who Sean Chen is.)